.github/workflows/build.yml

```
@@ -610,6 +610,13 @@ jobs:
           overwrite: true
           retention-days: 7
 
+      - name: Upload test results to Codecov
+        if: ${{ !cancelled() }}
```

```suggestion
        if: cancelled() == false
```

I think that's enough here?